Globes.js

```diff
@@ -0,0 +1,94 @@
+{
+	"translatorID": "4e9c34ef-89bb-4c7e-97dd-abb61d8b9254",
+	"label": "Globes",
+	"creator": "Eran Rosenthal",
+	"target": "^https?://www\\.globes\\.co\\.il/news/article.aspx",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-01 10:49:38"
+}
+
+/**
+	Copyright (c) 2019 Eran Rosenthal
+	
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	return 'newspaperArticle';
+}
+function doWeb(doc, url) { 
+	var item = new Zotero.Item('newspaperArticle');
+	
+	item.title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	item.publicationTitle = 'Globes';
+	item.url = url;
+	item.language = 'he';
```

Comment:
Can you also handle the English version of the Globes, e.g. https://en.globes.co.il/en/article-will-competition-authority-approve-taboola-outbrain-merger-1001302929 ?

Reply:
Done